frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/ticketView/ticketView.component.tsx

```diff
@@ -70,59 +81,162 @@ export const TicketView = ({
+	};
+
+	// State
+	const deleteGroups = () => {
+		const { state, ...view } = value || {};
+		onChange?.(view);
+	};
+
 	useEffect(() => { setTimeout(() => { onBlur?.(); }, 200); }, [value]);
 
 	const imgSrc = getImgSrc(value?.screenshot);
+	const hasCamera = !!value?.camera;
+
 	return (
-		<BasicTicketImage
-			value={imgSrc}
-			onChange={onImageChange}
-			disabled={disabled}
-			{...props}
-		>
-			{/* //TODO - delete after refactoring */}
-			<TicketImageAction onClick={onGroupsClick}>
-				<b>GROUPS</b>
-			</TicketImageAction>
-			<TicketImageAction onClick={goToViewpoint} disabled={disabled || !(value?.camera)}>
-				<GotoViewpointIcon />
-				<FormattedMessage id="viewer.card.ticketView.action.gotToViewpoint" defaultMessage="Go to viewpoint" />
-			</TicketImageAction>
-			{ !!(value?.camera) && (
-				<ActionMenu TriggerButton={(
-					<TicketImageAction disabled={disabled}>
-						<EditViewpointIcon />
-						<FormattedMessage id="viewer.card.ticketView.action.editViewpoint" defaultMessage="Edit viewpoint" />
-					</TicketImageAction>
-				)}
+		<InputContainer disabled={disabled} {...props}>
+			<Header>
+				<Label>{label}</Label>
+				<HeaderSection>
+					{!hasViewpoint ? (
+						<Tooltip
+							placement="right"
+							title={(formatMessage({ id: 'viewer.card.button.saveCurrentView', defaultMessage: 'Save current view' }))}
+						>
+							<div hidden={disabled}>
+								<PrimaryTicketButton onClick={updateViewpoint}>
+									<TickIcon />
+								</PrimaryTicketButton>
+							</div>
+						</Tooltip>
+					) : (
+						<Tooltip
+							placement="right"
+							title={(formatMessage({ id: 'viewer.card.button.gotToView', defaultMessage: 'Go to view' }))}
+						>
+							<div>
+								<PrimaryTicketButton onClick={goToViewpoint}>
+									<ViewpointIcon />
+								</PrimaryTicketButton>
+							</div>
+						</Tooltip>
+					)}
+					<EllipsisMenu disabled={!hasViewpoint}>
+						<EllipsisMenuItem
+							hidden={!hasViewpoint}
+							title={(<FormattedMessage id="viewer.card.ticketView.action.updateViewpoint" defaultMessage="Update to current view" />)}
+							onClick={updateViewpoint}
+							disabled={disabled}
+						/>
+						<EllipsisMenuItem
+							hidden={!hasViewpoint}
+							title={(<FormattedMessage id="viewer.card.ticketView.action.gotToView" defaultMessage="Go to view" />)}
+							onClick={goToViewpoint}
+						/>
+						<EllipsisMenuItemDelete
+							hidden={!hasViewpoint}
+							title={<FormattedMessage id="viewer.card.ticketView.action.deleteView" defaultMessage="Delete view" />}
+							onClick={deleteViewpoint}
+							disabled={disabled}
+						/>
+					</EllipsisMenu>
+				</HeaderSection>
+			</Header>
+			<TicketImageContent
+				value={imgSrc}
+				onChange={onImageChange}
+				disabled={disabled}
+			>
+				{/* Image */}
+				<TicketImageActionMenu value={imgSrc} onChange={onImageChange} disabled={disabled} />
+				{/* Camera */}
+				<ViewActionMenu
+					disabled={!hasCamera}
```

Comment:
Can we split all the actionMenus into different components like we did with ticketImageActionMenu.
Could be cameraActionMenu, groupsActionMenu